### PR TITLE
Header and footer with new font

### PR DESCRIPTION
**Pull request by fhzno**

### CODE REVIEW

File: src/components/footer.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ const Footer: React.FC = () => (
           <img
             src={logo}
             alt='logo'
-            style={{ marginRight: '55%', width: '55px', paddingLeft: '8' }}
+            style={{ width: '55px', paddingLeft: '8' }}
```
Comment (kali-maha) [inline src/components/footer.tsx]: We could use the premade classes from tailwindcss here 

also i think you forgot `px` on padding left :)

File: public/index.html [hunk 2]
```diff
@@ -9,6 +9,14 @@
...
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>Flow Protocol</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:ital@0;1&family=IBM+Plex+Sans:ital,wght@0,100;0,400;0,700;1,100;1,400;1,700&display=swap"
+      rel="preload" rel="stylesheet" as="style"
+    />
+    <link
+    href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:ital@0;1&family=IBM+Plex+Sans:ital,wght@0,100;0,400;0,700;1,100;1,400;1,700&display=swap"
+    rel="stylesheet"
+  />
```
Comment (kali-maha) [inline public/index.html]: I think we are using only 2 or 3 weights? would be nice if we dont have to load all of them plus italics

File: src/components/footer.tsx (hunk 1 above)
Comment (wytman) [reply src/components/footer.tsx]: Nice catch.

File: public/index.html (hunk 2 above)
Comment (wytman) [reply public/index.html]: Think we just need 400 and 500 maybe, but not sure.

Comment (wytman) [reply public/index.html]: Perhaps best to continue designing and revisit this and remove unused weights and italics.

**Review by kali-maha — APPROVED** [review]

**Pull request merged**